packages/realm/src/schema/validate.ts

```diff
@@ -149,6 +148,7 @@ export function validatePropertySchema(
   objectName: string,
   propertyName: string,
   propertySchema: unknown,
+  primaryKey: string | undefined,
```

Comment:
Does this make sense? @kraenhansen @elle-j @takameyer

Reply:
Currently we have the validation part (i.e. `validate.ts`) be responsible for checking that the JS data types are correct. E.g. checking that `objectType` is a string, but not checking that `objectType` only is set when `type` is e.g. `"list"` (that's done in `normalize.ts`). So the validation is just for making sure that other functions can rely on the types being what TS says it is.

So maybe we can move the logic of checking that the combination of `"full-text"` and other props are allowed, to `normalize.ts`?

Reply:
Makes total sense!